src/Configuration/NodeDefinition/TableNodesDecorator.php

```diff
@@ -18,15 +18,25 @@ public function addNodes(NodeBuilder $builder): void
             ->scalarNode('name')
                 ->cannotBeEmpty()
             ->end()
-            ->scalarNode('query')->end()
+            ->scalarNode('query')
+                ->cannotBeEmpty()
+            ->end()
             ->arrayNode('table')
                 ->children()
-                    ->scalarNode('schema')->isRequired()->end()
-                    ->scalarNode('tableName')->isRequired()->end()
+                    ->scalarNode('schema')
+                        ->cannotBeEmpty()
```

Comment:
u tohodle potom pozor při nasazování - doteď mohlo být prázdný, tak to nějakým klientům může začít padat
nemyslím jen schéma, ale obecně u všeho kde si přidal `cannotBeEmpty`

Reply:
Ano toho som si vedomý, no nejak to treba posunut dalej a budem na to mysliet pri nasadzovani a kontrolovat to.

Ak by si mal empty string v tych klucoch co som osetril, tak vacsinou by ti to spadlo niekde v kode - alebo by to nevratilo ziadny vysledok.

Napr. prazdna schema by ti tu bud spadla alebo nevratila ziadny vysledok: https://github.com/keboola/db-extractor-common/blob/058a8bb956cf4e84f362475d78755b97816b9adc/src/Extractor/Common.php#L139-L142 ... bolo by tam `SELECT FROM ""."tableName"` (z hlavy neviem co MySQL vrati) .... teda nemyslim si, ze to bude nejaky casty problem (kedze ani predtym by to neslo), ale nieco sa moze vyskytnut.